HopsanGUI/LogDataHandler2.cpp

```diff
@@ -927,7 +963,12 @@ void LogDataHandler2::importFromPlainColumnCsv(QString importFilePath, const QCh
         for (int i=1; i<data.size(); ++i)
         {
             SharedVariableDescriptionT pVarDesc = SharedVariableDescriptionT(new VariableDescription);
-            pVarDesc->mDataName = "CSV"+QString::number(i);
+            if(!names.isEmpty()) {
+                pVarDesc->mDataName = names[i];
```

Comment:
Assumes number of names same as number of columns, wiĺl crash if (out of bounds)  if this is not the case, (bad csv file)

if (names.size() > i) would likely fix this problem

Reply:
True, I will add a fix.